example/home/blocks.py

```diff
@@ -89,6 +92,24 @@ class TextAndButtonsBlock(blocks.StructBlock):
     ]
 
 
+@register_streamfield_block
+class TextWithCallableBlock(blocks.StructBlock):
+    text = blocks.CharBlock()
+
+    graphql_fields = [
+        GraphQLString("text"),
+        GraphQLField("string_property", graphene.String, source="get_string_property"),
+        GraphQLField("string_method", graphene.String, source="get_string_method"),
+    ]
+
+    @property
+    def get_string_property(self, *args, **kwargs):
+        return "A simple string property."
+
+    def get_string_method(self, *args, **kwargs):
+        return "A simple string method."
```

Comment:
Most real-life use cases would use this in combination with some other local value, so it would be great to have this (e.g. return the `text` block value with a prefix, or perhaps with spaces removed)